lib/cartopy/io/shapereader.py

```diff
@@ -131,9 +131,9 @@ class BasicReader:
 
     """
 
-    def __init__(self, filename, bbox=None):
+    def __init__(self, filename, bbox=None, encoding='utf-8'):
```

Comment:
I don't love that the defaults between the two readers isn't the same here... I think the `**kwargs` solution referenced by @lgolston would help by only sending along the encoding when a user explicitly sets it without needing to worry about the default between Fiona and Pyshp.